inlong-agent/agent-installer/conf/installer.properties

```diff
@@ -18,21 +18,21 @@
 #######################
 #   common config
 #######################
-local.ip=127.0.0.1
+agent.local.ip=127.0.0.1
 agent.enable.oom.exit=false
 
 ############################
 # manager config
 ############################
-manager.addr=http://127.0.0.1:8083
-manager.auth.secretId=
-manager.auth.secretKey=
+agent.manager.addr=http://127.0.0.1:8083
+agent.manager.auth.secretId=
+agent.manager.auth.secretKey=
 
 ############################
 # cluster config for automatically report and register
 ############################
-cluster.tag=default_cluster
-cluster.name=default_agent
+agent.cluster.tag=default_cluster
+agent.cluster.name=default_agent
 
 ############################
 # audit config
```

Comment:
plz unify all configurations

Reply:
In fact, because the configuration here will also be used by the agent later, it was intentionally changed to be the same as the agent

Reply:
I saw all the configurations besides `audit.enable` and `audit.proxys` are prefixed by `agent.`.
why do these two configurations not need the prefix?

Reply:
forgot

Reply:
Two points, 1. These two configurations are not exclusive to agents; 2 agents also use this configuration name